packages/core/test/VDocument.test.ts

```diff
@@ -1223,5 +1228,644 @@ describe('stores', () => {
+                        });
+                        // Backward selection
+                        testEditor({
+                            contentBefore: '<p>ab]cd[</p>',
+                            stepFunction: enter,
+                            contentAfter: '<p>ab</p><p>[]<br></p>',
+                        });
+                    });
+                    it('should delete all contents of a paragraph, then split it', () => {
+                        // Forward selection
+                        testEditor({
+                            contentBefore: '<p>[abcd]</p>',
+                            stepFunction: enter,
+                            contentAfter: '<p><br></p><p>[]<br></p>',
+                        });
+                        // Backward selection
+                        testEditor({
+                            contentBefore: '<p>]abcd[</p>',
+                            stepFunction: enter,
+                            contentAfter: '<p><br></p><p>[]<br></p>',
+                        });
+                    });
+                });
+            });
+            describe('With SHIFT key', () => {
+                describe('Range collapsed', () => {
+                    describe('Basic', () => {
+                        it('should insert a <br> into an empty paragraph', () => {
+                            testEditor({
+                                contentBefore: '<p>[]<br></p>',
+                                stepFunction: shiftEnter,
+                                // That range is equivalent to <br>[]<br>
+                                contentAfter: '<p><br><br>[]</p>',
+                            });
+                            testEditor({
+                                contentBefore: '<p>[<br>]</p>',
+                                stepFunction: shiftEnter,
+                                // That range is equivalent to <br>[]<br>
+                                contentAfter: '<p><br><br>[]</p>',
+                            });
+                            testEditor({
+                                contentBefore: '<p><br>[]</p>',
+                                stepFunction: shiftEnter,
+                                // That range is equivalent to <br>[]<br>
+                                contentAfter: '<p><br><br>[]</p>',
+                            });
+                        });
+                        it('should insert a <br> at the beggining of a paragraph', () => {
+                            testEditor({
+                                contentBefore: '<p>[]abc</p>',
+                                stepFunction: shiftEnter,
+                                contentAfter: '<p><br>[]abc</p>',
+                            });
+                            testEditor({
+                                contentBefore: '<p>[] abc</p>',
+                                stepFunction: shiftEnter,
+                                // The space should have been parsed away.
```

Comment:
Why ?

Reply:
Because it's at the beginning of a paragraph.